crates/miden-testing/src/kernel_tests/mod.rs

```diff
@@ -354,49 +351,7 @@ fn executed_transaction_account_delta_new() {
     );
 }
 
-#[test]
-fn test_empty_delta_nonce_update() {
-    let tx_script_src = "
-        use.test::account
-        begin
-            push.1
-
-            call.account::incr_nonce
-            # => [0, 1]
-
-            drop drop
-            # => []
-        end
-    ";
-
-    let tx_script = TransactionScript::compile(
-        tx_script_src,
-        TransactionKernel::testing_assembler_with_mock_account(),
-    )
-    .unwrap();
-
-    let tx_context = TransactionContextBuilder::with_standard_account(ONE)
-        .tx_script(tx_script)
-        .build();
-
-    // expected delta
-    // --------------------------------------------------------------------------------------------
-    // execute the transaction and get the witness
-    let executed_transaction = tx_context.execute().unwrap();
-
-    // nonce delta
-    // --------------------------------------------------------------------------------------------
-
-    // nonce was incremented by 1
-    assert_eq!(executed_transaction.account_delta().nonce(), Some(Felt::new(1)));
-
-    // storage delta
-    // --------------------------------------------------------------------------------------------
-    // We expect one updated item and one updated map
-    assert_eq!(executed_transaction.account_delta().storage().values().len(), 0);
-
-    assert_eq!(executed_transaction.account_delta().storage().maps().len(), 0);
-}
+// TODO add a test that updates the storage but does not increment the nonce
```

Comment:
Leftover?

Reply:
Ah indeed, this is covered by `test_epilogue_increment_nonce_violation`